python/phylanx/ast/transformation.py

```diff
@@ -10,9 +10,12 @@
 import phylanx
 import inspect
 from phylanx.exceptions import InvalidDecoratorArgumentError
-from phylanx.util import prange
 from .oscop import OpenSCoP
 from .utils import full_name, full_node_name, physl_fmt
+try:
+    from _phylanx import init_hpx_runtime
+except:
+    from _phylanxd import init_hpx_runtime
```

Comment:
It might be a good idea to separate the initialization of HPX from the transformation code. Should this go into a separate module instead?